acquisitions/src/main/resources/thunderjet/consortia/features/pieces-api-test-for-cross-tenant-envs.feature

```diff
@@ -0,0 +1,523 @@
+Feature: Pieces API tests for cross-tenant envs
+
+  Background:
+    * url baseUrl
+    * call login consortiaAdmin
+
+    * def targetTenant = universityTenant;
```

Comment:
keeping actual tenant name is more better.

Reply:
I will directly work with tenant and will specify headers that give more context